template-processors/base/bin/resourceManager.sh

```diff
@@ -53,15 +53,20 @@ function createUpdateResources {
 
 }
 
-echo Managing Resources
+if [ $CREATE_MODE == "None" ]; then
```

Comment:
I might be missing something, but how are you handling ResourceDeletionMode = None?

Reply:
@Smiley73 you're right I wasn't. Pushed another commit.

Reply:
@etsauer of course I was right!
My apologies if I haven't explained this to you previously, so here it is:
1. Heiko is always right
2. You are never right
3. In case you should be right, #1 automatically applies.

Reply:
Thanks for enlightening me.